contrib/openshift/skydive-template.yaml

```diff
@@ -202,6 +204,74 @@ objects:
         - hostPath:
             path: /var/lib/origin
           name: lib-origin
+# Skydive FlowExporter (Deployment)
```

Comment:
could it be a dedicated template file adding the support of flow exporter ?

Reply:
@safchain done.